hazelcast-jet-distribution/src/assembly/zip.xml

```diff
@@ -53,6 +54,23 @@
             <outputDirectory>config</outputDirectory>
         </file>
 
+        <!-- Default YAML config files -->
+        <file>
+            <source>${config.directory}/hazelcast-jet-default.yaml</source>
+            <destName>hazelcast-jet.yaml</destName>
+            <outputDirectory>config</outputDirectory>
+        </file>
+        <file>
+            <source>${config.directory}/hazelcast-jet-member-default.yaml</source>
+            <destName>hazelcast.yaml</destName>
+            <outputDirectory>config</outputDirectory>
```

Comment:
better to put them under config/examples, otherwise it's not clear which config files are loaded.